pkg/authorization/workspace_content_authorizer.go

```diff
@@ -128,17 +114,8 @@ func (a *workspaceContentAuthorizer) Authorize(ctx context.Context, attr authori
 	}
 
 	switch {
-	case isServiceAccount && isForeign:
```

Comment:
We remove this so default is just "out of scope"?

Reply:
Default is to go through normal access authorization. Only exception: local service account have access automatically.